### PR TITLE
SHEL-165 Animate component and transition modifiers

### PR DESCRIPTION
**Pull request by willeeklund**
New features:
* A new component called Animate. This is what will be used primarily.
* A transition modifier for animating height. This is what Animate uses internally.

This zip contains a recording of how the animation looks.
[animate_component.zip](https://github.com/nordnet/nordnet-ui-kit/files/1025360/animate_component.zip)

Screenshot from the new documentation:
image

### CODE REVIEW

File: src/components/animate/animate.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+This is how you should use the Animate component.
+
+```html
+<Animate type="height" show={variableToDecideIfShown} enterTime={300} leaveTime={200}>
```
Comment (bstream) [inline src/components/animate/animate.md]: Remove `show` here

File: src/styles/transitions/index.js [hunk 2]
```diff
@@ -1,25 +1,8 @@
 // Easings and durations from material design, see their docs for usage information:
 // https://material.google.com/motion/duration-easing.html#duration-easing-natural-easing-curves
```
Comment (bstream) [inline src/styles/transitions/index.js]: Might want to move these comments to their new locations respectively

File: src/components/animate/animate.md (hunk 1 above)
Comment (willeeklund) [inline src/components/animate/animate.md]: Fixed now.

File: src/styles/transitions/index.js (hunk 2 above)
Comment (willeeklund) [inline src/styles/transitions/index.js]: Fixed now.

**Pull request merged**